### PR TITLE
Add logo icons to network switcher

### PR DESCRIPTION
**Pull request by n13**
### 🗃 Github Issue Or Explanation for this PR. (What is it supposed to do and Why is needed)

Adding the chains to the network switcher

Adding images and logos, loading logos from github polkadot JS
### ✅ Checklist

- [ ] Github issue details are up to date for people to QA.
- [x] I have tested all my changes.

### 🙈 Screenshots
image

### CODE REVIEW

File: scripts/balance.js [hunk 1]
```diff
@@ -24,7 +24,10 @@ const init = async () => {
 
   console.log(JSON.stringify(chain, null, 2))
 
-  await getLastBlock(api)
+  //await getLastBlock(api)
```
Comment (n13) [inline scripts/balance.js:27]: unrelated change just affecting scripts - ignore

File: lib/components/chain_avatar.dart [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      child: Container(
+        width: size,
+        height: size,
+        child: _buildContent(context),
+      ),
+    );
+  }
+
+  Widget _buildContent(BuildContext context) {
+    if (image != null) {
```
Comment (gguijarro-c-chwy) [inline lib/components/chain_avatar.dart:34]: all of this logic can be moved away from the UI into the bloc.

File: lib/screens/profile_screens/switch_network/interactor/usecases/get_network_data_use_case.dart [hunk 3]
```diff
@@ -29,6 +30,55 @@ final mockData = [
...
+        }
+      } else {
+        for (final devChain in chainContainer.list ?? []) {
+          final icon = await chainsRepository.resolveIcon(devChain.info);
+          list.add(devChain.toNetworkData(icon));
+        }
+      }
+    }
+
+    return Future.delayed(const Duration(milliseconds: 300)).then((value) => Result.value(list));
```
Comment (gguijarro-c-chwy) [inline lib/screens/profile_screens/switch_network/interactor/usecases/get_network_data_use_case.dart:82]: do we need delayed here?

File: lib/components/chain_avatar.dart (hunk 2 above)
Comment (n13) [reply lib/components/chain_avatar.dart:34]: But which block? This should be a reusable component that can show any sort of image, svg, asset, png, web... 

I don't want to have to care about what asset it is, and I don't want to put that in every bloc that uses it.

Comment (n13) [reply lib/components/chain_avatar.dart:34]: I just wanna show an image, IMO that's what the normal image widget could do it it wanted to, dump any kind of resource on it, and it works.

File: lib/screens/profile_screens/switch_network/interactor/usecases/get_network_data_use_case.dart (hunk 3 above)
Comment (n13) [reply lib/screens/profile_screens/switch_network/interactor/usecases/get_network_data_use_case.dart]: Whoops!!!

Comment (n13) [reply lib/screens/profile_screens/switch_network/interactor/usecases/get_network_data_use_case.dart]: Removed.

**Comment by n13** [comment]
Need to move forward with this, but will find solution for the avatar widget in another PR @gguijarro-c-chwy

File: lib/components/chain_avatar.dart (hunk 2 above)
Comment (gguijarro-c-chwy) [reply lib/components/chain_avatar.dart:34]: if (image!.startsWith("http")) {
        if (image!.endsWith(".svg")) {
          return SvgPicture.network(image!, fit: BoxFit.cover);
        } else {
          return CachedNetworkImage(imageUrl: image!, fit: BoxFit.cover);
        }
      } else if (image!.startsWith("asset")) {
        if (image!.endsWith(".svg")) {
          return SvgPicture.asset(image!, fit: BoxFit.scaleDown);
        } else {
          return Image.asset(image!);
        }
      }
      
      ALl these stuff can be  moved away from the UI. You are doing logic in the UI and this should be avoided. 
      
"But which block? This should be a reusable component that can show any sort of image, svg, asset, png, web..."
Then have the reusable component take a Widget instead of an Image, this way you can pass in there anything you want. 

It is not up to the UI component to determine if its HTTP or asset or w.e, the UI component should be dumb, you give me a widget i render a widget.

Comment (gguijarro-c-chwy) [reply lib/components/chain_avatar.dart:34]: its fine here, this is a small scale project and this wont change. But when we write UI components, there is always a way to move most if not all logic away from them.
